src/convenience/session.ts

```diff
@@ -384,8 +384,7 @@ class PropertySession<O extends {}, P extends keyof O> {
             enumerable: true,
             get: () => {
                 if (key === undefined) {
-                    const msg = undef("access", opts);
-                    throw new Error(msg);
+                    return undefined;
```

Comment:
This now causes a type mismatch, see https://github.com/grammyjs/grammY/issues/713#issuecomment-2694864398